ui/API_scripts.js

```diff
@@ -1,119 +1,120 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
I'd view this while ignoring whitespace:
https://github.com/epic-power-rpg/roll20/pull/9/files?diff=unified&w=1

This is mostly auto-formatted via linter.